MerchantSDK/GiniMerchantSDKExample/GiniMerchantSDKExample/Sources/InvoicesList/InvoiceTableViewCellModel.swift

```diff
@@ -31,12 +31,12 @@ final class InvoiceTableViewCellModel {
         return ""
     }
     
-    var dueDateText: String {
-        invoice.paymentDueDate ?? ""
+    var ibanText: String {
+        invoice.iban ?? ""
     }
     
     var isDueDataLabelHidden: Bool {
```

Comment:
[Clarification]: can we remove isDueDataLabelHidden since the field was removed already?